### PR TITLE
HV: remove unused API lookp_entry_by_id()

### PR DESCRIPTION
**Pull request by zhijianli88**
Signed-off-by: Li Zhijian <lizhijian@cn.fujitsu.com>

### CODE REVIEW

**Comment by NanlinXie** [comment]
Can one of the admins verify this patch?

File: hypervisor/arch/x86/assign.c [hunk 1]
```diff
@@ -69,7 +69,7 @@ _lookup_entry_by_id(uint32_t id)
 }
 
 static inline struct ptdev_remapping_info *
-lookp_entry_by_id(uint32_t id)
+lookup_entry_by_id(uint32_t id)
```
Comment (dongyaozu) [inline hypervisor/arch/x86/assign.c]: Good Catch @zhijianli88 
But, why this is an orphan API ?  @gzhai7

Comment (edwinzhai) [reply hypervisor/arch/x86/assign.c]: Sorry. It's a mistake when spliting the big patches. This orphan API will be removed by the 2nd patches.  I'll continue to split the 2nd into small ones to address this.

Leave it as is or delete it. I'll handle it in next small patches.

Comment (dongyaozu) [reply hypervisor/arch/x86/assign.c]: In this case, @zhijianli88 please delete the API.

**Comment by lijinxia** [comment]
Verified+1

**Pull request merged**